src/app/files/file-explorer.js

```diff
@@ -209,7 +209,9 @@ function fileExplorer (localRegistry, files, menuItems) {
     const provider = self._deps.fileManager.fileProviderOf(key)
     actions['Create File'] = () => self.createNewFile(key)
     actions['Create Folder'] = () => self.createNewFolder(key)
-    if (provider.isExternalFolder(key)) {
+    // todo not fully implemented. Readd later when fixed
```

Comment:
Just to make it easier to find for VSCode todo extensions could you write it like that `@todo(#2386) not fully ...`